mkdocs.yml

```diff
@@ -86,6 +86,13 @@ nav:
           - GAMS:       Documentation/Applications/gams.md
           - Gaussian:   Documentation/Applications/gaussian.md
           - Gurobi:     Documentation/Applications/gurobi.md
+          - IDAES Solvers : 
```

Comment:
Can you add the new applications to the Applications table as well: https://nrel.github.io/HPC/Documentation/Applications/